azurerm/internal/services/securitycenter/resource_arm_security_center_workspace.go

```diff
@@ -63,19 +61,6 @@ func resourceArmSecurityCenterWorkspaceCreateUpdate(d *schema.ResourceData, meta
 
 	name := securityCenterWorkspaceName
 
-	if features.ShouldResourcesBeImported() && d.IsNewResource() {
-		existing, err := client.Get(ctx, name)
-		if err != nil {
-			if !utils.ResponseWasNotFound(existing.Response) {
-				return fmt.Errorf("Error checking for presence of existing Security Center Workspace: %+v", err)
-			}
-		}
-
-		if existing.ID != nil && *existing.ID != "" {
-			return tf.ImportAsExistsError("azurerm_security_center_workspace", *existing.ID)
-		}
-	}
-
```

Comment:
Incase the default is removed and a new one created we need to keep this check

Reply:
The api `Create` works exactly as `Update` works, which means that if we duplicated create this resource, it'll just update this resource. Thus, could we remove the duplication check?